apps/server/src/app.seed.ts

```diff
@@ -0,0 +1,17 @@
+import { createApp } from '@/app.factory';
+
+import { UserSeed } from '@/user/seed/user.seed';
+
+async function seed() {
+  const app = await createApp();
+
+  /**Seed data
+   * Seeds the user to <SEED_USER_EMAIL]> if not present and assign default role to the user
+   */
+  const userSeed = app.get(UserSeed);
+
+  //seed user
+  await userSeed.seed();
+}
```

Comment:
Maybe pass the `app` as function parameter instead of creating a new app.

Reply:
Agreed. But, need to pass the app parameter when calling every other seed if created. 

Maybe for straight forward seed creation. createApp() shouldn't be an issue.
```
async function seed() {
  const app = await createApp();

  /**Seed data
   * Seeds the user to <SEED_USER_EMAIL]> if not present and assign default role to the user
   */
  const userSeed = app.get(UserSeed);

  //seed user
  await userSeed.seed();
}
```

minimizes the use of 
`await seed()` in `main.ts` . Isn't it? @SamipSthaLF  Need some insights for this.

Reply:
I'm not sure creating multiple instances of app is the way to go. App should be singleton.

Reply:
I would have to agree with yachu on this one 

while this is a seed that will be eventually removed later on, creating multiple instances of an app in the server may put an unforeseen load on the system and we would eventually need a system to seed 

as our primary goal for this project is to set standards for leapfrog itself it would be better if we start with using and passing around the same instance of the app lets make sure that the main app.main has no clutter though

Reply:
Updated to pass `INestApplication` as param for seed